bin/annotate_test_failures

```diff
@@ -63,13 +68,23 @@ class TestFailure
 
   def to_s
     times = @count > 1 ? " (#{@count} times)" : ''
+
+    # Do not render a code block for the failure details if there are none
+    formatted_details = if @details.strip.empty?
+                          ''
+                        else
+                          <<~DETAILS
+
+                            ```
+                            #{@details}
+                            ```
+                          DETAILS
+                        end
+
     <<~ENTRY
       <details><summary><tt>#{@name}</tt> in <tt>#{@classname}</tt>#{times}</summary>
-      #{@message}
-
-      ```
-      #{@details}
-      ```
+      <pre>#{@message}</pre>
+      #{formatted_details}
```

Comment:
I could be convinced that the `<pre>` makes sense only when there are no details.

In case both failure and details have a value, we'd get something like:

`failure name`

```
details details
details
```

That might not be the best to look at 🤔 

Then again, we don't have to look at that info for long 😅